### PR TITLE
added pvc annotations

### PR DESCRIPTION
**Pull request by proserie**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/unifi/templates/pvc.yaml [hunk 1]
```diff
@@ -3,6 +3,11 @@ kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: {{ template "unifi.fullname" . }}
+  {{- if .Values.persistentAnnotations }}
+  annotations:
+    {{- range $key, $value := .Values.persistentAnnotations }}
+  {{ $key }}: {{ $value | quote }}
```
Comment (mcronce) [inline stable/unifi/templates/pvc.yaml]:
```suggestion
    {{ $key }}: {{ $value | quote }}
```

File: stable/unifi/templates/pvc.yaml [hunk 2]
```diff
@@ -3,6 +3,11 @@ kind: PersistentVolumeClaim
...
+  {{- end }}
```
Comment (mcronce) [inline stable/unifi/templates/pvc.yaml:11]:
```suggestion
    {{- end }}
  {{- end }}
```

File: stable/unifi/values.yaml [hunk 3]
```diff
@@ -297,3 +297,5 @@ affinity: {}
 podAnnotations: {}
 
 deploymentAnnotations: {}
+
+persistantAnnotations: {}
```
Comment (mcronce) [inline stable/unifi/values.yaml]:
```suggestion
persistentAnnotations: {}
```

File: stable/unifi/templates/pvc.yaml [hunk 4]
```diff
@@ -21,4 +26,4 @@ spec:
   storageClassName: "{{ .Values.persistence.storageClass }}"
 {{- end }}
 {{- end }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
```
Comment (mcronce) [inline stable/unifi/templates/pvc.yaml]:
```suggestion
```

**Review by mcronce — CHANGES_REQUESTED** [review]
@proserie I _think_ this will work with the changes I've requested.  Once you make them, please add DCO signoff to your commits and I'll tell the CI robot to run tests.

Thanks for the addition!

File: stable/unifi/templates/pvc.yaml (hunk 4 above)
Comment (proserie) [reply stable/unifi/templates/pvc.yaml]: I tested this one and it broke so I left this one as is. This end is for the if statement in line 1

**Comment by proserie** [comment]
Closing since it is deprecated

**Pull request closed without merging**